### ISSUE TITLE
Version 1.3.1 is not marked as 'latest' on NPM

### ISSUE THREAD

**Issue by rjbailey** [HubSpot/tether#165]
I'd like to get the `scrollParent` fix included in 1.3.1 via a transitive dependency in react-tether, which depends on tether `^1.1.1`. Unfortunately, version 1.3.0 is marked as `latest` on NPM, so installing react-tether installs tether 1.3.0.

`npm view` shows that the `latest` dist-tag and `version` are still at 1.3.0--can you get it updated?

```
> npm view tether

{ name: 'tether',
  versions:
   [ '0.5.2',
     '0.6.5',
     '0.7.1',
     '0.7.2',
     '1.0.0',
     '1.0.1',
     '1.0.2',
     '1.0.3',
     '1.1.0',
     '1.1.1',
     '1.2.0',
     '1.2.2',
     '1.2.3',
     '1.2.4',
     '1.3.0',
     '1.3.1' ],
  'dist-tags': { latest: '1.3.0' },
  version: '1.3.0',
```

**Comment by ChrisPearce** [comment]
npm view tether is now showing latest as v1.2.4 (rather than 1.3.1)

`{ name: 'tether',
   description: 'A client-side library to make absolutely positioned elements attach to elements in the page efficiently.',
   'dist-tags': { latest: '1.2.4' },
   versions: 
   [ '0.5.2',
     '0.6.5',
     '0.7.1',
     '0.7.2',
     '1.0.0',
     '1.0.1',
     '1.0.2',
     '1.0.3',
     '1.1.0',
     '1.1.1',
     '1.2.0',
     '1.2.2',
     '1.2.3',
     '1.2.4',
     '1.3.0',
     '1.3.1' ],
`

**Comment by geekjuice** [comment]
Should be fixed. `npm` will point the last published version (in our case `v1.2.4` which was mistakenly not published at that time) as `latest` despite being older in terms of semver.
